handbook/benefits.md

```diff
@@ -53,26 +78,69 @@ The following stipend amounts have been developed to help cover some of the cost
 
 ### Receiving the Benefits Stipend
 
-The benefits stipend will automatically be added to your monthly payroll. You do not need to request for a reimbursement through Airbase.
+The benefits stipend will automatically be added to your monthly payroll. You do not need to request a reimbursement through Airbase.
```

Comment:
Above - Line 28/53 - can we change to "How does this apply today?"

Line 40/65 change "**L**ife" to "**l**ife"